avocado/utils/disk.py

```diff
@@ -121,15 +121,19 @@ def get_disks():
     :returns: a list of paths to the physical disks on the system
     :rtype: list of str
     """
-    json_result = process.run("lsblk --json --paths --inverse")
-    json_data = json.loads(json_result.stdout_text)
-    disks = []
-    for device in json_data["blockdevices"]:
-        disks.append(device["name"])
-        if "children" in device:
-            for child in device["children"]:
-                disks.append(child["name"])
-    return disks
+    try:
+        json_result = process.run("lsblk --json --paths --inverse")
+        json_data = json.loads(json_result.stdout_text)
+        disks = []
+        for device in json_data["blockdevices"]:
+            disks.append(device["name"])
+            if "children" in device:
+                for child in device["children"]:
+                    disks.append(child["name"])
+        return disks
+    except Exception as e:
+        print(f"Error occurred while getting disk information: {e}")
+        return []
```

Comment:
> I'm not sure this is the best behavior for this function, based on the other functions in the same module. They all raise `DiskError` on error conditions, instead of "silencing" the exception. I'll leave that for other users and writers of these API to comment on.

The adjustment has been made according to your opinion, please help to review it